Sources/ArgumentParser/Parsable Properties/NameSpecification.swift

```diff
@@ -38,10 +38,10 @@ public struct NameSpecification: ExpressibleByArrayLiteral {
     /// Short labels can be combined into groups.
     case customShort(Character)
   }
-  var elements: Set<Element>
+  var elements: Array<Element>
   
   public init<S>(_ sequence: S) where S : Sequence, Element == S.Element {
-    self.elements = Set(sequence)
+    self.elements = Array(sequence)
```

Comment:
It's great that these will be in a consistent order, although we want to preserve the uniqueness we were getting from `Set`. This fails the command validation, but could just be collapsed into a single `.short` name:

```swift
struct Example: ParsableCommand {
  @Option(name: [.short, .short])
  var example: Int
}
```

Reply:
I will add this case to tests.